### ISSUE TITLE
Unblock me 

### ISSUE THREAD

**Issue by Sudhindhra0202** [tachiyomiorg/tachiyomi-extensions#11377]
### Source information

All

### Steps to reproduce

1.when i open a extension it is showing loading it is not showing and action 

### Expected behavior

Plzzz help me 

### Actual behavior

Plzzz solve my problem as soon as possible 

### Tachiyomi version

Latest update

### Android version

Android 12

### Other details

Keep up the good work 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by loocool2** [comment]
Please follow the terms laid out in the acknowledgements. Begging to be unblocked with next to no other information will not allow us to help. Follow the troubleshooting guide as well before making an issue.

**Comment by CarlosEsco** [comment]
you just earned a block on github
